src/test/test.js

```diff
@@ -1,7 +1,101 @@
 var Oncoprint = require("../../dist/oncoprint.bundle.js");
+var BucketSort = require("../js/bucketsort.js");
+var assert = require("chai").assert;
 
 describe("test", function() {
-  it("should have oncoprint object", function() {
-    expect(Oncoprint).toBeDefined();
-  });
+    it("should have oncoprint object", function() {
+        assert.isDefined(Oncoprint);
+    });
+});
+
+describe("bucketSort", function() {
+    describe("bucketSort", function() {
```

Comment:
need empty list test

also, use randomly generated tests and compare sorted order vs the order given by BucketSort.compare

also, tests with infinity

also, tests with compareEquals